### ISSUE TITLE
underscore.js dependency for a single extend call is overkill

### ISSUE THREAD

**Issue by dominikg** [silvermine/videojs-airplay#17]
This is a runtime dependency and has to be downloaded, extracted, parsed by every enduser.

Please replace the extend call with Object.assign or some other plain method and drop the dependency.

**Comment by dominikg** [comment]
see https://github.com/silvermine/videojs-airplay/blob/305c61346434e007207c07c8d1c93e40f2998644/src/js/enableAirPlay.js#L96

**Comment by jthomerson** [comment]
Good point. Underscore is just a standard library we use a lot of places, but you're right we only have the one call in this codebase. Pull requests welcome. Make sure to familiarize yourself with our [coding standards](https://github.com/silvermine/silvermine-info/blob/master/coding-standards.md) in order to ensure your PR success.

**Comment by yokuze** [comment]
Please also note that our plugins support IE 11, so whatever solution is chosen must be compatible with IE 11, which `Object.assign` [is not](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Browser_compatibility).

**Comment by dominikg** [comment]
the most simple solution would be like this:

```
var pluginOptions =  options || {};
if(pluginOptions.addButtonToControlBar == null ) {
  pluginOptions.addButtonToControlBr = true;
}
```

this changes the behavior slighly (mutation of passed options if addButtonToControlBar isn't defined)

**Comment by jthomerson** [comment]
Fixed by #34
